apps/docs/src/.vitepress/config.ts

```diff
@@ -35,6 +35,13 @@ export default defineConfig({
     ],
     ["meta", { property: "og:site_name", content: "onyx" }],
     ["meta", { property: "og:image", content: "https://onyx.schwarz/images/og-logo.webp" }],
+    ["meta", { property: "og:image:type", content: "image/webp" }],
+    ["meta", { property: "og:image:width", content: "1200" }],
+    ["meta", { property: "og:image:height", content: "630" }],
+    ["meta", { property: "og:image", content: "https://onyx.schwarz/images/og-logo.jpg" }],
+    ["meta", { property: "og:image:type", content: "image/jpeg" }],
+    ["meta", { property: "og:image:width", content: "1200" }],
+    ["meta", { property: "og:image:height", content: "630" }],
```

Comment:
Don't they override each other because they have the same property name?

Reply:
Maybe its enough to just have the jpg version and remove webp?

Reply:
According to https://ogp.me/#array you can have a property multiple times.